site.properties

```diff
@@ -90,10 +90,19 @@ security.subscribe.enable = true
 #jpa.showSql          = false
 #jpa.generateDDL      = false
 
+# LDAP
+
+ldap.providerURL        = ldaps://centaur.unimelb.edu.au
+ldap.userContext        = ou=people,o=unimelb
+ldap.usernameAttribute  = uid
+ldap.emailAttribute     = mail
+ldap.firstNameAttribute = givenName
+ldap.lastNameAttribute  = sn
+
 # Database and JPA (MySQL)
 
 jdbc.driver   = com.mysql.jdbc.Driver
-jdbc.url      = jdbc:mysql://localhost/apromore?autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
+jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
```

Comment:
I was wondering if we should let liquibase do the creation and have an account that has permissions to create database instead.

Reply:
Ideally yes, For now Kept this the same and intention is to get up and runnig without hasle of setup

Reply:
Why H2 and MySQL settings are separated by LDAP in-between?

Reply:
I think this came in merge commit, Will change the positioning.

Reply:
Done.